Content.Shared/Humanoid/Prototypes/SpeciesPrototype.cs

```diff
@@ -164,4 +164,5 @@ public enum SpeciesNaming : byte
     FirstLast,
     FirstDashFirst,
     TheFirstofLast,
+	LastFirst,
```

Comment:
CD marker comment needed.

Reply:
SS14 also uses spaces, not tabs. Do you have editorconfig set up for your editor?